src/deepsparse/benchmark/benchmark_model.py

```diff
@@ -391,14 +390,15 @@ def benchmark_model(
         "num_streams": num_streams,
         "benchmark_result": benchmark_result,
     }
+    parsed_export_dict = {**model._properties_dict(), **export_dict}
```

Comment:
First populate kwargs in _properties_dict(), if there is an intersection in export_dict, because its values are closer to the source of truth (straight from the input args, instead of variable from class -- can be mutated by mistake), overwrite it.